lib/tasks/set_duplicate_references.rake

```diff
@@ -1,7 +1,7 @@
 desc "Set duplicate references for sales and lettings logs"
 task set_duplicate_references: :environment do
   SalesLog.filter_by_year(2023).duplicate_sets.each do |duplicate_set|
-    duplicate_set_id = generate_new_duplicate_set_id
+    duplicate_set_id = SalesLog.maximum(:duplicate_set_id).to_i + 1
```

Comment:
do you think it's worth making this a function so we can describe what it does (and as we're doing it in 4 places)

Reply:
hmm, so I think the it's only here, in the controller we've got it in the `new_duplicate_set_id` method.
I'm not sure if there's a good place to put this for a rake task and given we should only have to run it once, I assume this would get deleted relatively quickly anyways